File: Java/1001-2000/1701-1800/1771-1780/1773.java
class Solution {
    public int countMatches(List<List<String>> items, String ruleKey, String ruleValue) {
        int count = 0, rvk = 0;
        if (ruleKey.equals("color"))
            rvk = 1;
        else if (ruleKey.equals("name"))
            rvk = 2;
        for (List<String> l : items) {
            if ((l.get(rvk)).equals(ruleValue)) {
                count++;
            }
        }
        return count;
    }
}

File: Python/1001-2000/1701-1800/1771-1780/1773.py
class Solution:
    def countMatches(self, items: List[List[str]], ruleKey: str, ruleValue: str) -> int:
        i = 0
        c = 0
        if ruleKey == "color":
            i = 1
        elif ruleKey == "name":
            i = 2
        for l in items:
            if l[i] == ruleValue:
                c += 1
        return c
